change/@internal-react-components-0fbe78f6-c749-429e-9950-68f078c9a8b1.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Create a device ermissions sub directory",
```

Comment:
ermissions => permissions

Reply:
🤦 you are correct but since this is a `none` changelog and I am being lazy and not going onto the branch to make the change I'm going to leave this